.gitignore

```diff
@@ -1,6 +1,5 @@
 node_modules
 npm-debug.log
 typings
-dist
```

Comment:
@tobiasmuecksch Im a bit new to publishing modules on npm, do we need to include the dist folder inside the repo? Or should I leave it ignored?